requirements.txt

```diff
@@ -9,9 +9,11 @@ numpy==1.20.1
 plaidml-keras==0.7.0
 psutil==5.8.0
 pyserial==3.5
+python-socketio==5.3.0
```

Comment:
Remove `python-socketio` and `requests` they will be in `optional-requirements.txt`

Reply:
Oh, sorry -- I thought I did this already, but apparently I didn't. Thanks for pointing it out!

I ended up amending the latest commit, as this is something that should have been included in that commit, already.